main.js

```diff
@@ -0,0 +1,6 @@
+const oComponentBoilerplate = require('./src/js/componentBoilerplate');
+
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
It needs to listen to the `o.DOMContentLoaded` event instead of dispatching it https://github.com/Financial-Times/o-tabs/blob/master/main.js#L3